docs/src/hugo/content/index.md

```diff
@@ -357,11 +362,76 @@ namespaces:
 
 Here the two units are similar, but the second defines a `schedule` under the plans stanza, which indicates that the unit is to be run periodically as a `job`.
 
-The manifest contains a variety of settings and options too numerous to mention in this introductory text. Suggested further reading [in the reference](reference.html#manifest) about the Nelson manifest answers the following common queries:
+<h3 id="user-guide-manifest-workflow-blueprints" class="linkable">Workflows & Blueprints</h3>
+
+The *Nelson* manifest enables the user to focus primarily on their own application and not worry about the specifics of runtime scheduling. Whilst this can be largely beneficial, it also presents a set of challenges as more often than not every organization has a different internal process with slight differences. To accommodate this, *Nelson* supports the concept of `Workflow` and `Blueprint`. 
+
+A `Blueprint` is a scheduler specification (for example, Kubernetes YAML document(s) or Nomad HCL) mustache template, which at deploy-time *Nelson* will interpolate with the runtime data (e.g. `StackName`), and submit that rendered output to the scheduler API in order to launch a stack. Whilst the author would hope that this powerful tool is carefully used, it provides the operators / admins of *Nelson* the flexibility to create deployment stacks that use whatever runtime setup they want. For example, Company X might wish to run their logging and routing as sidecars, with Kubernetes pods using particular node affinity, where as Company Y might want to instead just use node selectors for deployments and rely on some node global configuration. *Nelson* simply does not care.
```

Comment:
May be useful to give a concrete example here - for instance `StackName` doesn't really appear in the manifest. One example might be how health checks are transformed into a Kubernetes [Liveness Probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) and how the `retries` field is interpolated into Kubernetes `backoffLimit`.